components/style/properties/gecko.mako.rs

```diff
@@ -614,9 +614,13 @@ def set_gecko_property(ffi_name, expr):
             }
         };
         match length {
-            Either::First(number) =>
-                self.gecko.${gecko_ffi_name}.set_value(CoordDataValue::Factor(From::from(number))),
-            Either::Second(lop) => self.gecko.${gecko_ffi_name}.set(lop),
+            SvgLengthOrPercentageOrNumber::LengthOrPercentage(lop) => {
+                // TODO: Stylo treat calc value, but gecko side doesn't support yet.
+                // https://bugzilla.mozilla.org/show_bug.cgi?id=1386967
```

Comment:
This comment doesn't belong here. This will be fixed on the Gecko-side when it starts to handle the calc type for these properties.